config/sync/image.style.redia_feed_large.yml

```diff
@@ -0,0 +1,15 @@
+uuid: 923e70e6-ea34-446d-ae06-2f6ac0ac4f00
+langcode: en
+status: true
+dependencies: {  }
+name: redia_feed_large
+label: 'Redia feed large'
+effects:
+  71f3eb2a-c6f8-4a85-a16f-651a6e43239e:
+    uuid: 71f3eb2a-c6f8-4a85-a16f-651a6e43239e
+    id: image_scale
+    weight: 1
+    data:
+      width: 2000
```

Comment:
:+1: for adding a max width. It seems as if the current implementation does not really set one but I think it is a good thing that we do now.